packages/cli/src/commands/install.ts

```diff
@@ -17,12 +16,11 @@ export async function install() {
     console.log("✅ Done!")
     return
   } catch (error) {
-    if (error.isTtyError) {
+    if (error instanceof Error) {
       console.warn(
         "Prompt couldn't be rendered in the current environment, exiting..."
       )
+      throw error
```

Comment:
I'm confused why we don't rethrow the error here. If it was thrown once we can throw it again, no?
Without modification to the `error` object that should guarantee it's the same error object so we can perform error operations on it.

Reply:
I'm tempted to completely deprecate this installer. We don't encourage its use anywhere.

Reply:
I'll do it in another PR after this one is merged.